backend/app/crud/base.py

```
@@ -0,0 +1,70 @@
+from typing import Any, Dict, Generic, List, Optional, Type, TypeVar, Union
```

Ruff (0.14.5)

`typing.Dict` is deprecated, use `dict` instead

(UP035)